components/button/theme.css

```diff
@@ -35,7 +35,7 @@
   vertical-align: middle;
 
   svg {
-    margin: auto;
+    margin: auto 0; /* fix for vertical centering in safari */
```

Comment:
we added the `margin: auto` as a fix for vertical centering in safari, but the `auto` was messing horizontal alignment up in IE. `margin: auto 0` fixes it everywhere. Everywhere.

 :ok: by @lowiebenoot on 13/11/2017 17:19:42